en/starter/generator.md

```diff
@@ -35,16 +35,15 @@ Swift Server Generator can create the [scaffolding](/{{ page.lang }}/starter/gen
 
 ## Tutorials
 
-### [Connecting to a Bluemix Cloudant service](/{{ page.lang }}/starter/generator/cloudant_tutorial_bluemix.html)
+### [Connecting to a Cloudant service](/{{ page.lang }}/starter/generator/cloudant_tutorial_bluemix.html)

```

Comment:
Maybe rename Bluemix in the link as well?

Reply:
done

Reply:
done by neil